conda/models/records.py

```diff
@@ -387,6 +384,18 @@ def record_id(self):
         return "%s/%s::%s-%s-%s" % (self.channel.name, self.subdir,
                                     self.name, self.version, self.build)
 
+    @property
+    def metadata(self) -> str:
+        # join all metadatas after filtering out noops
+        return " ".join(
+            filter(
+                None,
+                [
+                    signature_verification(self),
+                ],
```

Comment:
This can be refactored pretty nicely using the plugin framework:

```python
@conda.plugins.register
def conda_metadata():
    return signature_verification
```

```suggestion
                pm.hook.conda_metadata(),
```